### DISCUSSION TITLE
How to create multiple blank pages?

### DISCUSSION THREAD

**Discussion by victorfeagins** [quarto-dev/quarto-cli#10953] [Q&A] [answered]
### Description

I am interested in making worksheets that involve blank pages for students to work on.  Using the shortcode `{{< pagebreak >}}` seems to only work if there is content after the page break. The work around I've been doing is including a little bit of text after the page break. Is there an easy way to get multiple page breaks or blank pages?

````
---
title: "test"
format: pdf
---

## Quarto

Quarto enables you to weave together content and executable code into a finished document. To learn more about Quarto see <https://quarto.org>.

{{< pagebreak >}}

{{< pagebreak >}}

````

**Comment by arnaudgallou** [comment]
You can simply add a back slash in between the two page breaks:

```
---
title: "test"
format: pdf
---

## Quarto

Quarto enables you to weave together content and executable code into a finished document. To learn more about Quarto see <https://quarto.org>.

{{< pagebreak >}}

\

{{< pagebreak >}}
```

**Reply by mcanouil** [reply]
FYI, `\` at the end of a line means "linebreak" in Markdown.

**Reply by victorfeagins** [reply, marked as answer]
Thanks for the input! The small adjustment is the `\` after the second page break as well.
````
---
title: "test"
format: pdf
---

## Quarto

Quarto enables you to weave together content and executable code into a finished document. To learn more about Quarto see <https://quarto.org>.

{{< pagebreak >}}

\

{{< pagebreak >}}

\
````
